app/main/index.js

```diff
@@ -358,6 +358,19 @@ app.on('ready', () => {
 	ipcMain.on('save-last-tab', (event, index) => {
 		ConfigUtil.setConfigItem('lastActiveTab', index);
 	});
+
+	// Update user idle status for each realm after every 15s
+	// Set user idle if no activity for idleThreshold time in seconds
+	setInterval(() => {
+		const idleThreshold = 60;
```

Comment:
The 60s and 15s seem to not agree?

Reply:
Oh, never mind; might want to make the 15000 as a `const` variable too for readability.

Reply:
Will change.